### PR TITLE
Feat: Adding Heroku and AWS Lambda documentation

### PR DESCRIPTION
**Pull request by tkadlec**
This adds documentation for Heroku and AWS under two new sections.

Still going to give it more prominent display on the landing page, but in this PR I'm keen to get some verification that I haven't made things up. :)

To test, bring the branch local, hit jekyll serve and read away!

### CODE REVIEW

File: _docs_aws-lambda/5_adding-snyk-specific-user.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: Adding a Snyk-specific user to AWS Lambda
+---
+We recommend adding a dedicated AWS Identity and Access Management (IAM) user for your Snyk organization. That way if at some point you need to revoke the key for any reason, you can do so without impacting anyone within your organization.
+
+The IAM user only needs one role: `AWSLambdaReadOnlyAccess`.
```
Comment (gjvis) [inline _docs_aws-lambda/5_adding-snyk-specific-user.md]: The IAM user only needs one ~role~ attached policy: `AWSLambdaReadOnlyAccess`.

File: _docs_aws-lambda/6_disabling-aws-lambda.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+---
+If you decide to disable the AWS Lambda integration for any reason, you can accomplish this from the ["Integration Settings"](https://snyk.io/org/snyk/manage/integrations) page in your settings.
+
+You'll need to find the AWS Lambda integration in your list of integrations, and click "Edit Settings". You'll be taken to a page that shows the current status of your AWS Lambda connection, a place to update your API key, and a red box at the bottom to disconnect from AWS Lambda.
+
+![Screenshot showing the Disconnect screen for disabling the AWS Lambda integration](http://res.cloudinary.com/snyk/image/upload/c_scale,q_auto,w_auto/v1493156542/serverless-docs/aws-disconnect.png)
+
+If you choose to disconnect, your AWS Lambda credentials will be removed from Snyk and any AWS Lambda projects we had been monitoring will be deactivated on Snyk.
+
+If you choose to re-enable the AWS Lambda integration at any time, you'll need to re-enter your credentials and active your projects.
```
Comment (gjvis) [inline _docs_aws-lambda/6_disabling-aws-lambda.md]: active -> activate

File: _docs_aws-lambda/00_intro.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+title: ""
+---
+
+Snyk's AWS Lambda integration lets you monitor the deployed code of your Node.js and Ruby AWS Lambda functions for any known vulnerabilities found in the application's dependencies, testing at a frequency you control.
```
Comment (gjvis) [inline _docs_aws-lambda/00_intro.md]: No ruby support on AWS Lambda (because AWS doesn't support Ruby)

File: _docs_heroku/6_disabling-heroku.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+---
+If you decide to disable the Heroku integration for any reason, you can accomplish this from the ["Integration Settings"](https://snyk.io/org/snyk/manage/integrations) page in your settings.
+
+You'll need to find the Heroku integration in your list of integrations, and click "Edit Settings". You'll be taken to a page that shows the current status of your Heroku connection, a place to update your API key, and a red box at the bottom to disconnect from Heroku.
+
+![Screenshot showing the Disconnect screen for disabling the Heroku integration](http://res.cloudinary.com/snyk/image/upload/c_scale,q_auto,w_auto/v1493154598/serverless-docs/heroku-disconnect.png)
+
+If you choose to disconnect, your Heroku credentials will be removed from Snyk and any Heroku projects we had been monitoring will be deactivated on Snyk.
+
+If you choose to re-enable the Heroku integration at any time, you'll need to re-enter your credentials and active your projects.
```
Comment (gjvis) [inline _docs_heroku/6_disabling-heroku.md]: active -> activate

File: _docs_aws-lambda/00_intro.md (hunk 3 above)
Comment (tkadlec) [inline _docs_aws-lambda/00_intro.md]: Copy/paste issue. My bad. :)

File: _docs_aws-lambda/5_adding-snyk-specific-user.md (hunk 1 above)
Comment (tkadlec) [inline _docs_aws-lambda/5_adding-snyk-specific-user.md]: Fixed

File: _docs_aws-lambda/6_disabling-aws-lambda.md (hunk 2 above)
Comment (tkadlec) [inline _docs_aws-lambda/6_disabling-aws-lambda.md]: Fixed

File: _docs_heroku/6_disabling-heroku.md (hunk 4 above)
Comment (tkadlec) [inline _docs_heroku/6_disabling-heroku.md]: Fixed

**Pull request merged**